### DISCUSSION TITLE
How to disable type checking for nuxt layers?

### DISCUSSION THREAD

**Discussion by avxkim** [nuxt/nuxt#29974] [Questions]
I have parent app and a child layer.

nuxt.config.ts (parent):

```
  typescript: {
    typeCheck: true,
  },
```

child is located at "layers/child-layer" and it has tons of ts errors, how do i ignore "child-layer" errors in a parent app?

**Comment by angelhdzmultimedia** [comment]
Did you try setting typeCheck: false for that layer?

**Reply by avxkim** [reply]
This would disable type checking for layer itself, i mean the case, where i start an app that consumes child-layer, because i want to see errors only related to this app itself, not child-layer

**Reply by angelhdzmultimedia** [reply]
Oh, so you want the layer to have typecheck if you open the layer separately to work with it, but at the same time, typechecking being disabled when you load it in your main app to hide the errors? 🤔 Interesting... subscribing to wait for a solution...

"Disable typechecking for imported layers".
